twilight-model/src/guild/role_tags.rs

```diff
@@ -25,6 +25,14 @@ pub struct RoleTags {
         with = "crate::visitor::null_boolean"
     )]
     pub premium_subscriber: bool,
+
+    /// Whether this role is a guild's linked role.
+    #[serde(
+        default,
+        skip_serializing_if = "is_false",
+        with = "crate::visitor::null_boolean"
+    )]
+    pub guild_connections: bool,
```

Comment:
we alphabetically sort our struct members; this includes instantiations (e.g. tests)